doc/design/staging-1.md

```diff
@@ -30,12 +30,22 @@ To test code changes to the web GUI, branch previews will also be pointed at the
 
 The production web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
 
-* A developer has a feature they want to add that impacts both the backend and the frontend.
+A developer has a feature they want to add that impacts both the backend and the frontend.
 * They checkout a new branch in `dandi-api` and `dandiarchive`, do the work, and submit related PRs in both repos.
```

Comment:
both against `master` branch? (it is better to state branches at this step since PR needs to know the base branch to start with, so better to state here and remove branches mentioning below for "PR is merged into ...")

Reply:
I have removed any reference in this file to any other branch except `master`, so I think it would be implicit to anyone reading the document. If anyone is confused about where to base the PR when following these instructions, I don't think it's unreasonable for them to read one step ahead for the answer.